types/chrome/index.d.ts

```diff
@@ -2074,11 +2087,11 @@ declare namespace chrome.declarativeContent {
 declare namespace chrome.declarativeWebRequest {
     export interface HeaderFilter {
         nameEquals?: string | undefined;
-        valueContains?: any;
+        valueContains?: string | string[];
         nameSuffix?: string | undefined;
         valueSuffix?: string | undefined;
         valuePrefix?: string | undefined;
-        nameContains?: any;
+        nameContains?: string | string[];
```

Comment:
Since the `exactOptionalPropertyTypes` typescript flag is set, undefined should be explicitly declared.
```suggestion
        nameContains?: string | string[] | undefined;
```